tasks/release.rb

```diff
@@ -44,13 +44,21 @@
       raise "Could not insert PRE in #{file}" unless $1
 
       File.open(file, 'w') { |f| f.write ruby }
+
+      if File.exist?("#{framework}/package.json")
+        Dir.chdir("#{framework}") do
+          version = version.gsub(/\./).with_index { |s, i| i >= 2 ? '-' : s }
+          sh "npm version #{version} --no-git-tag-version"
+        end
+      end
     end
 
     task gem => %w(update_versions pkg) do
       cmd = ""
       cmd << "cd #{framework} && " unless framework == "rails"
       cmd << "bundle exec rake package && " unless framework == "rails"
       cmd << "gem build #{gemspec} && mv #{framework}-#{version}.gem #{root}/pkg/"
+      cmd << "&& npm publish" if File.exist?("#{framework}/package.json")
```

Comment:
Building the gem should not publish it to NPM. No more than it should push the gem to RubyGems. This is a _build_ task, not a _publish_ task.